### PR TITLE
feat: self-hosting

### PR DESCRIPTION
**Pull request by slavovojacek**

### CODE REVIEW

**Review by jeremyletang — APPROVED** [review]
looks good to me!

File: go.sum [hunk 1]
```diff
@@ -3,6 +3,7 @@ cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMT
 cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
 cloud.google.com/go v0.44.1/go.mod h1:iSa0KzasP4Uvy3f1mN/7PiObzGgflwredwwASm/v6AU=
 cloud.google.com/go v0.44.2/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxKY=
+cloud.google.com/go v0.44.3/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxKY=
```
[the remark this answers is not in the document]
Comment (slavovojacek) [reply go.sum:6]: Thanks, that's done now!

**Comment by slavovojacek** [comment]
https://github.com/Homebrew/homebrew-core/pull/96851/files

**Comment by slavovojacek** [comment]
https://github.com/Homebrew/homebrew-core/pull/96851

**Pull request merged**